assets/styles/app.scss

```diff
@@ -47,6 +46,10 @@ header {
 
 // Navigation Bar pour le fichier _navigation.html.twig
 
+footer {
+    margin-top: 40%;
```

Comment:
this doesn't look like a good spacing (always 1rem ;) ). I guess you should handle this through flexbox